pom.xml

```diff
@@ -19,7 +19,7 @@
 
     <groupId>org.eclipse.microprofile</groupId>
     <artifactId>microprofile-parent</artifactId>
-    <version>1.0</version>
+    <version>2.0</version>
```

Comment:
This version is bumped manually instead of by the CI/CD bot? At least, that's how it works on the pom.xml for the platform spec, for example. Do we already have instructions somewhere that provide instructions for this?